### PR TITLE
Initial commit (Sha256HmacKey2019 context).

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
From https://github.com/digitalbazaar/webkms-context/pull/1

### CODE REVIEW

File: js/context.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+module.exports = {
+  '@context': {
+    id: '@id',
+    type: '@type',
+    '@protected': true
+
```
Comment (dlongley) [inline js/context.js:11]: This doesn't have the `Sha256HmacKey2019` term defined.

Comment (dmitrizagidulin) [reply js/context.js:11]: ::facepalms:: one sec..

Comment (dmitrizagidulin) [reply js/context.js:11]: Fixed.

**Review by dlongley — APPROVED** [review]
At a quick glance, seems ok.

**Pull request merged**